modules/wri_taxonomy/wri_taxonomy.install

```diff
@@ -38,3 +38,13 @@ function wri_taxonomy_install() {
 function wri_taxonomy_update_9501() {
   \Drupal::service('distro_helper.updates')->updateConfig('views.view.resource_types', ['display#default#display_options#arguments'], 'wri_taxonomy');
 }
+
+/**
+ * Add taxonomy term topics and subtopics.
+ */
+function wri_taxonomy_update_9502() {
+  \Drupal::service('distro_helper.updates')
+    ->installConfig('field.field.taxonomy_term.topics_and_subtopics.field_default_image.yml', 'wri_taxonomy');
```

Comment:
You'll need to leave off the `.yml` here (and on the next line). Right now I'm getting:
`[error]  Config file not found at profiles/contrib/wri_sites/modules/wri_taxonomy/config/install/field.field.taxonomy_term.topics_and_subtopics.field_default_image.yml.yml`

This may be a thing that changed between versions of distro_helper.

Reply:
fixed.